apl-core/src/main/resources/META-INF/beans.xml

```diff
@@ -1,9 +1,13 @@
 <?xml version="1.0"?>
 <beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+       xmlns:weld="http://jboss.org/schema/weld/beans"
        xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
        version="1.1" bean-discovery-mode="all">
   <decorators>
     <class>org.jboss.weld.environment.se.threading.RunnableDecorator</class>
   </decorators>
+  <weld:scan>
```

Comment:
What is a reason to exclude it?

Reply:
What was attempt to fix error by me, when 'fat jar' is created and there was one error that can be fixed by that xml. Looks like that doesn't effect the node run.

Reply:
Now 'beans.xml' is updated a lot. There are many excludes in it for 'fat jar'.